app/src/main/res/layout-land/option_audio_language.xml

```diff
@@ -23,27 +23,25 @@
 
     <TextView
       android:id="@+id/audio_language_label_text_view"
+      style="@style/Subtitle1"
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
-      android:fontFamily="sans-serif"
       android:text="@string/audio_language"
       android:textAlignment="viewStart"
       android:textColor="@color/oppiaPrimaryTextDark"
-      android:textSize="16sp"
       app:layout_constraintBottom_toTopOf="@id/audio_language_text_view"
       app:layout_constraintEnd_toEndOf="parent"
       app:layout_constraintStart_toStartOf="parent"
       app:layout_constraintTop_toTopOf="parent" />
 
     <TextView
       android:id="@+id/audio_language_text_view"
+      style="@style/Subtitle2"
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
```

Comment:
Remove `layout_height` attribute as it is already mentioned in `Body` style. (Make sure its `wrap_content` only in styles.xml)